guide/xml/using.xml

```diff
@@ -998,6 +998,62 @@ Warning: no license set
                 </varlistentry>
             </variablelist>
         </section>
+
+        <section xml:id="using.port.load">
+            <title>port load</title>
+
+            <para>Some ports install software that is meant to run as a daemon. Or in other words, a long-running
+                    background process.</para>
+
+            <para>Examples of this are database servers like MySQL or PostgreSQL.</para>
+
+            <para>On macOS, <command>launchd</command> is primarily responsible for starting, stopping, and
+                    managing long-running services.</para>
+
+            <para>Ports that want to run daemon processes can install their own <filename>.plist</filename> file(s)
+                    into <command>launchd</command>. These files will allow <command>launchd</command> to start and
+                    manage the port's daemon processes.</para>
+
+            <para>So once a port is installed, the <option>load</option> action can be used to do the above
+                    and activate the port's <command>launchd</command> service(s):</para>
+
+            <programlisting><prompt>$ </prompt><userinput>sudo port load prometheus</userinput></programlisting>
+
+            <screen>
+---&gt;  Loading startupitem 'prometheus' for prometheus</screen>
+
+            <para>Now the port's service(s) should be running in <command>launchd</command>. This can be verified with
+                    the <command>launchctl</command> command:</para>
+
+            <programlisting><prompt>$ </prompt><userinput>sudo launchctl list | grep macports</userinput></programlisting>
+
+            <screen>
+49119   0       org.macports.prometheus</screen>
+
+            <para>To stop the daemon service permanently and remove the port from <command>launchd</command>, use 
+                    the <link linkend="using.port.unload"><command>port unload</command></link> command.</para>
+
+        </section>
+
+        <section xml:id="using.port.unload">
+            <title>port unload</title>
+
+            <para>As discussed in the <link linkend="using.port.load"><command>port load</command></link> section, the
+                    <command>port load</command> command can be used to install and activate a port's daemon service(s) in
+                    <command>launchd</command>.</para>
+
+            <para>The <option>unload</option> action reverses this.</para>
+
+            <para><command>port unload</command> will stop the port's daemon processes and remove the port's service <filename>.plist</filename>
+                    file(s) from <command>launchd</command>:</para>
```

Comment:
This could be a bit misleading, as the plist file itself is not removed from anywhere when `port unload` is run. The job is just marked as disabled.

Reply:
> This could be a bit misleading, as the plist file itself is not removed from anywhere when `port unload` is run. The job is just marked as disabled.

That's great feedback.

So in the interest of progress, can you suggest alternate wording, etc, so that this can be wrapped up?

Reply:
> This could be a bit misleading, as the plist file itself is not removed from anywhere when `port unload` is run. The job is just marked as disabled.

Agreed; wording changed as appropriate.